<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>591</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1148</width>
    <height>591</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mixer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
padding-left: 2px
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QLabel" name="compLabel_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>AlHor</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Component 1:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="imageLabel_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>AlHor</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Image 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="6" colspan="2">
     <widget class="QGraphicsView" name="image_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>210</width>
        <height>210</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="6" column="4" colspan="2">
     <widget class="QSlider" name="slider_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="2">
     <widget class="QComboBox" name="mixerComp_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>31</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Phase</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uniform Phase</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="actionImage_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Open Image</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QGraphicsView" name="comp_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>210</width>
        <height>210</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="6">
     <widget class="QGraphicsView" name="comp_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>210</width>
        <height>210</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="mixerLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>155</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>AlHor</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mixer output to:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QComboBox" name="singleComp_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>25</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Choose Component</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magnitude</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Phase</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Real</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Imaginary</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QComboBox" name="mixerImage_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>31</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Image 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Image 2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QGraphicsView" name="image_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>210</width>
        <height>210</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="mixerImage_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>31</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Image 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Image 2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="3" alignment="Qt::AlignRight">
     <widget class="QLabel" name="sliderValue_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>100 %</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="compLabel_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>AlHor</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Component 2:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QComboBox" name="setOutput">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>31</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Choose Output</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Output 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Output 2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QSlider" name="slider_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2" alignment="Qt::AlignBottom">
     <widget class="QComboBox" name="singleComp_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>25</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Choose Component</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magnitude</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Phase</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Real</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Imaginary</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignBottom">
     <widget class="QPushButton" name="actionImage_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Open Image</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignBottom">
     <widget class="QLabel" name="imageLabel_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>AlHor</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Image 1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" alignment="Qt::AlignRight">
     <widget class="QLabel" name="sliderValue_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>100 %</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QComboBox" name="mixerComp_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>31</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Magnitude</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Phase</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Real</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Imaginary</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uniform Magnitude</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uniform Phase</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QGraphicsView" name="output_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>210</width>
        <height>210</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="interactive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QGraphicsView" name="output_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>210</width>
        <height>210</height>
       </size>
      </property>
      <property name="interactive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QLabel" name="outputLabel_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>AlHor</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Output 2</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QLabel" name="outputLabel_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>AlHor</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Output 1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
